PublicationsRouter/v2/api/API.md

```diff
@@ -199,18 +205,64 @@ Any of the notification endpoints listed below will return one of these response
 
 Note the last of these is different from the Validation endpoint.
 
-- On **authentication failure** (e.g. invalid api_key, incorrect user role) the system will respond with a 401 (Unauthorised) and no response body.
+- On **authentication failure** (e.g. invalid api_key, incorrect user role) the system will respond with a **401 (Unauthorised)** and no response body.
+
+```
+    HTTP 1.1  401 Unauthorized
+```
+
+- In the event of a **malformed HTTP request**, the system will respond with a **400 (Bad Request)** and the response body:
 
-- In the event of a **malformed HTTP request**, the system will respond with a 400 (Bad Request) and the response body:
 ```
     HTTP 1.1  400 Bad Request
     Content-Type: application/json
-    
+
     {
+        "status": "error",
         "error" : "human readable error message"
     }
 ```
+
+- In the event of a **Not Acceptable HTTP request** for a 'notification list', the system will respond with a **406 (Not Acceptable)** and the response body:
+
+```
```

Comment:
You should use place-holders rather than values in the following, that would explain what the values mean.